src/pinwheel-group/pinwheel-group.style.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from "lit";
+
+export default css`
+  :host {
+    display: inline-flex;
+  }
+
+  .pinwheel-group {
+    position: relative;
+  }
+
+  .selector-indicator {
+    /* FIXME: we dont have sizing tokens yet */
+    height: 48px;
+    background-color: var(--tap-pinwheel-group-color-surface-tertiary, var(--tap-sys-color-surface-tertiary));
```

Comment:
I think `--tap-pinwheel-group-background-color` is a better name

```suggestion
    background-color: var(--tap-pinwheel-group-background-color, var(--tap-sys-color-surface-tertiary));
```